app/src/main/java/com/spoony/spoony/presentation/register/componet/PhotoPicker.kt

```diff
@@ -0,0 +1,242 @@
+                                        singlePhotoPickerLauncher.launch(
+                                            PickVisualMediaRequest(ActivityResultContracts.PickVisualMedia.ImageOnly)
+                                        )
+                                    } else {
+                                        photoPickerLauncher.launch(
+                                            PickVisualMediaRequest(ActivityResultContracts.PickVisualMedia.ImageOnly)
+                                        )
+                                    }
+                                } else {
+                                    galleryLauncher.launch("image/*")
+                                }
+                            }
+                        ),
+                    contentAlignment = Alignment.Center
+                ) {
+                    Column(
+                        horizontalAlignment = Alignment.CenterHorizontally,
+                        verticalArrangement = Arrangement.Center
+                    ) {
+                        Icon(
+                            imageVector = ImageVector.vectorResource(id = R.drawable.ic_plus_24),
+                            contentDescription = null,
+                            tint = SpoonyAndroidTheme.colors.gray400,
+                            modifier = Modifier.size(16.dp)
+                        )
+                        Text(
+                            text = "${selectedPhotosList.size}/$MAX_PHOTO_COUNT",
+                            style = SpoonyAndroidTheme.typography.caption1m,
+                            color = SpoonyAndroidTheme.colors.gray400
+                        )
+                    }
+                }
+            }
+
+            itemsIndexed(
+                selectedPhotosList,
+                key = { _, item -> item.id }
+            ) { index, photo ->
+                Box(
+                    modifier = Modifier.size(80.dp)
+                ) {
+                    AsyncImage(
+                        model = photo.uri,
+                        contentDescription = null,
+                        modifier = Modifier
+                            .fillMaxSize()
+                            .clip(RoundedCornerShape(8.dp)),
+                        contentScale = ContentScale.Crop
+                    )
+                    Icon(
+                        imageVector = ImageVector.vectorResource(id = R.drawable.ic_delete_filled_24),
+                        contentDescription = null,
+                        tint = SpoonyAndroidTheme.colors.gray400,
+                        modifier = Modifier
+                            .size(20.dp)
+                            .noRippleClickable {
+                                val newList = selectedPhotosList.toMutableList()
+                                newList.removeAt(index)
+                                onPhotosSelected(newList.toPersistentList())
```

Comment:
```suggestion
                                onPhotosSelected(
                                    selectedPhotosList.removeAt(index)
                                )
```

P1: 위에서 PersistentList로 바꿔주면 이렇게 바로 사용 가능합니다!

Reply:
selectedPhotosList: PersistentList<SelectedPhoto>

onPhotosSelected(selectedPhotosList.removeAt(index))
로 수정했습니다!